docs-ref-toc/reference-latest.yml

```diff
@@ -1813,7 +1813,7 @@
     href: ~/api/overview/azure/latest/search.md
     landingPageType: Service
     items:
-    - name: Cognitive Search
```

Comment:
The `reference-*.yml` files are updated by automation so these changes will be overwritten. Update the `DisplayName` field in this CSV file instead: 

https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv#L8